src/batch.rs

```diff
@@ -446,6 +452,18 @@ fn ddbjson_val_to_attrval(ddb_jsonval: &JsonValue) -> Option<AttributeValue> {
             ns: Some(set_logic(x)),
             ..Default::default()
         })
+    } else if let Some(x) = ddb_jsonval.get("BS") {
+        let binary_set = x
+            .as_array()
+            .expect("should be valid JSON array")
+            .iter()
+            .map(|el| Bytes::copy_from_slice(el.as_str().unwrap().as_bytes()))
```

Comment:
Based on the test code, we should pass the base64 decoded data into `Bytes::copy_from_slice` or a similar method.